src/test/scala/PropertyCheckerTest.scala

```diff
@@ -1,4 +1,5 @@
 package CSVValidation
+import Errors.MetadataError
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.databind.node._
 import org.scalatest.{FunSuite, Tag}
```

Comment:
Can you add some tests into here to ensure that datetime format validation works properly via the PropertyChecker?

Reply:
I have added 2 new tests which checks if DateFormat class works as planned with PropertyChecker.